torch/fft/__init__.py

```diff
@@ -865,6 +865,10 @@
 Reorders n-dimensional FFT data, as provided by :func:`~torch.fft.fftn`, to have
 negative frequency terms first.
 
+This performs a periodic shift of n-dimensional data such that the origin
+``(0, ..., 0)`` is moved to the center of the tensor. Specifically,
```

Comment:
Suggestion: 

"Specifically, ..." -> "Specifically, to ..." to make the second sentence parallel to the first.